dotnet/src/Connectors/Connectors.AzureAIInference/Settings/AzureAIInferenceChatExecutionSettings.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Text.Json;
+using Azure.AI.Inference;
+using Microsoft.SemanticKernel.Text;
+
+namespace Microsoft.SemanticKernel.Connectors.AzureAIInference;
+
+/// <summary>
+/// Chat completion execution settings.
+/// </summary>
+public class AzureAIInferenceChatExecutionSettings : PromptExecutionSettings
```

Comment:
The naming we using use would be `AzureAIInferencePromptExecutionSettings`, why is the pattern different here?

Reply:
The tendency is to move away from a name that is not per-modality. 
Each modality having a different setting.

- `{xyz}TextToImageExecutionSettings`
- `{xyz}ChatExecutionSettings`
- `{xyz}ImageToTextExecutionSettings`

Reply:
For chat we use `{xyz}PromptExecutionSettings` everywhere else. Was a decision made that we should change this pattern?

![image](https://github.com/user-attachments/assets/61eabdc7-1d82-46ec-b3c7-ae738585b39f)